3-enrich/scala-common-enrich/src/main/scala/com.snowplowanalytics.snowplow.enrich/common/utils/ConversionUtils.scala

```diff
@@ -457,6 +457,31 @@ object ConversionUtils {
     }
   }
 
+  /**
+   * Converts a String to a Double with two decimal places. Used to honor schemas with
+   * multipleOf 0.01.
+   * Takes a field name and a string value and return a validated double.
+   */
+  val stringToTwoDecimals: (String, String) => Validation[String, Double] = (field, str) =>
+    try {
+      BigDecimal(str).setScale(2, BigDecimal.RoundingMode.HALF_UP).toDouble.success
```

Comment:
Why not HALF_EVEN? Is this currency? HALF_EVEN has better statistical characteristics.

Reply:
yes this is used for currency-related fields

Reply:
In which case HALF_EVEN will produce more accurate results on aggregate

Reply:
changed to half even 👍